heron/packing/src/java/com/twitter/heron/packing/binpacking/FirstFitDecreasingPacking.java

```diff
@@ -114,10 +117,22 @@ public static String getComponentName(String instanceId) {
   @Override
   public void initialize(Config config, Config runtime) {
     this.topology = com.twitter.heron.spi.utils.Runtime.topology(runtime);
-
     this.instanceRamDefault = Context.instanceRam(config);
     this.instanceCpuDefault = Context.instanceCpu(config).doubleValue();
     this.instanceDiskDefault = Context.instanceDisk(config);
+
+    List<TopologyAPI.Config.KeyValue> topologyConfig = topology.getTopologyConfig().getKvsList();
+    this.maxContainerRam = Long.parseLong(TopologyUtils.getConfigWithDefault(
+        topologyConfig, com.twitter.heron.api.Config.TOPOLOGY_CONTAINER_MAX_RAM_HINT,
+        Long.toString(instanceRamDefault * 4)));
```

Comment:
4 here and below appears to be a magic number. What does it represent and should it be a constant or a config setting with a default?